backend/tests/v5/unit/schemas/tickets/templates.constants.test.js

```diff
@@ -15,12 +15,18 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+const { times, cloneDeep } = require('lodash');
 const { src } = require('../../../helper/path');
+const { generateRandomString } = require('../../../helper/services');
+const { statusTypes } = require('../../../../../src/v5/schemas/tickets/templates.constants');
 
 const TemplateConstants = require(`${src}/schemas/tickets/templates.constants`);
 
 const testGetApplicableDefaultProperties = () => {
 	describe('Get applicable default properties', () => {
+		const customStatusValues = times(5, () => ({ name: generateRandomString(), type: statusTypes[0] }));
```

Comment:
same here, make your test case use all possible status types

Reply:
Since you've need this in multiple place you may as well put it in the service helper?